pandas/_libs/lib.pyx

```diff
@@ -927,6 +929,8 @@ def indices_fast(ndarray index, const int64_t[:] labels, list keys,
 
 def is_float(obj: object) -> bool:
     """
+    Return a boolean if this is a float object.
```

Comment:
I think these are the same as above (`Return True`)

Reply:
changing docstrings to 

```
Return True if given object is [type].
```